neutrinoapp/src/Result.jsx

```diff
@@ -15,9 +15,17 @@ export default function Result () {
   const getData = async () => {
     setIsActive(true)
     try {
+      var resp = ""
       fetch('http://ec2-13-40-156-226.eu-west-2.compute.amazonaws.com:5000/api/moving-images-ent/entity/' + id)
         .then(response => response.json())
         .then(response => {
+          resp = response
+          return response.json()
+        })
+        .then(response => {
```

Comment:
I'm fuzzy on async things -- is it possible to evaluate the respond in the previous "then" block to avoid needing to introduce the "resp" variable?

Reply:
And perhaps also avoiding the need to explicitly return response.json()?

Reply:
Thanks for explaining. I see that resp seems to have a pretty limited scope, too, so it's not a big increase in complexity.